swift/apple/FirezoneKit/Sources/FirezoneKit/Models/WebAuthSession.swift

```
@@ -32,7 +32,13 @@ struct WebAuthSession {
         return
       }
 
-      Task { try await store.signIn(authResponse: authResponse) }
+      Task {
+        do {
+          try await store.signIn(authResponse: authResponse)
+        } catch {
+          Log.error(error)
```

Fixed to not silently swallow errors here.